seq2seq/models/DecoderRNN.py

```diff
@@ -166,7 +166,10 @@ def decode(step, step_output, step_attn):
 
             for di in range(decoder_output.size(1)):
                 step_output = decoder_output[:, di, :]
-                step_attn = attn[:, di, :]
+                if attn:
```

Comment:
Please use `if attn is not None`.  When `attn` is a tensor, it crushes because the boolean value of a tensor is ambiguous.